.github/workflows/build.yml

```
@@ -29,6 +29,12 @@ jobs:
       run: |
         dotnet tool install --global Microsoft.Playwright.CLI
 
+    - name: Configure BrowserStack credentials
+      if: ${{ github.repository_owner == 'martincostello' }}
```

Change this (maybe not exactly this way) to not configure credentials in PRs.

```suggestion
      if: ${{ github.repository_owner == 'martincostello' && github.ref == 'refs/heads/main' }}
```